cli.go

```diff
@@ -146,6 +158,37 @@ func (c *CLI) setupRateLimiter() {
 		c.c.String("rate-limit-prefix"))
 }
 
+func (i *CLI) setupOpenCensus(accountJSON string) error {
```

Comment:
Before we ship this out, it might be good to have a config setting to dynamically turn opencensus on or off.